packages/component-library/src/Slider/Slider.js

```diff
@@ -1,20 +1,22 @@
 import React from "react";
-import { bool, func, node, number } from "prop-types";
+import { bool, func, node, number, string, arrayOf } from "prop-types";
 import RcSlider from "rc-slider";
 import "rc-slider/assets/index.css";
 
 import "./slider.css";
 
 const SliderWithTooltip = RcSlider.createSliderWithTooltip(RcSlider);
-
+const Range = RcSlider.createSliderWithTooltip(RcSlider.Range);
 const Slider = ({
   min,
   max,
   onChange,
   showStepMarkers,
   step,
   value,
-  SliderComponent
+  SliderComponent,
+  sliderType,
+  defaultValue
```

Comment:
Can you remove the `sliderType` and `defaultValue` props from this file? They don't appear to be used anywhere and we want to avoid dead code.